docs/running-the-show/midi-dmx-or-audio-triggering.md

```diff
@@ -65,21 +67,22 @@ console to react in different ways and easily swap between them.
 
 This is how you set up a trigger:
 
-1. Add a new mapping using the + button at the bottom of the left hand
-column, and enter a name for it.
+1.  Add a new mapping using the \[+\] button at the bottom of the *left hand*
```

Comment:
As it's a soft button elsewhere we've just put the plus symbol in bold

Reply:
Okay - I'll follow this in my next commits

Reply:
I think it would make the instructions more obvious if this was rendered as a softkey as originally suggested. I think it would be OK to use the softkey rendering for any touch button on the screen - we do usually qualify the menu softkeys by saying "the [xxxx] softkey".  Also I think we have already used this rendering for context menu buttons in some places.

Reply:
100% agreed (honestly this is the approch I have been following in the german manual for quite a time now). From my experience the users not so much follow the distinction between a command hard key and a softkey, but much more separate between keys at all and _any_ command touch button in the display, be it a soft button, a context button, or a command button in any window.